src/main/java/FileNotFoundException.java

```diff
@@ -0,0 +1,2 @@
+public class FileNotFoundException extends Exception{
+}
```

Comment:
Refrain from keeping unused code since you're already using the default `FileNotFoundException` exception found in Java library.